sdk/keyvault/azure-keyvault-keys/samples/hello_world_async.py

```diff
@@ -53,28 +51,28 @@
 
         # Let's create an Elliptic Curve key with algorithm curve type P-256.
         # if the key already exists in the Key Vault, then a new version of the key is created.
-        print("\n1. Create an EC Key")
+        print("\n.. Create an EC Key")
         key_curve = "P-256"
         key_name = "ECKeyName"
         ec_key = await client.create_ec_key(key_name, curve=key_curve, hsm=False)
         print("EC Key with name '{0}' created of type {1}.".format(ec_key.name, ec_key.key_material.kty))
 
         # Let's get the rsa key details using its name
-        print("\n2. Get a Key using it's name")
+        print("\n.. Get a Key using it's name")
         rsa_key = await client.get_key(rsa_key.name)
         print("Key with name '{0}' was found.".format(rsa_key.name))
 
         # Let's say we want to update the expiration time for the EC key and disable the key to be usable
         # for cryptographic operations. The update method allows the user to modify the metadata (key attributes)
         # associated with a key previously stored within Key Vault.
-        print("\n3. Update a Key by name")
+        print("\n.. Update a Key by name")
         expires = datetime.datetime.utcnow() + datetime.timedelta(days=365)
         updated_ec_key = await client.update_key(ec_key.name, ec_key.version, expires=expires, enabled=False)
         print("Key with name '{0}' was updated on date '{1}'".format(updated_ec_key.name, updated_ec_key.updated))
         print("Key with name '{0}' was updated to expire on '{1}'".format(updated_ec_key.name, updated_ec_key.expires))
 
         # The RSA key is no longer used, need to delete it from the Key Vault.
-        print("\n4. Delete Key")
+        print("\n.. Delete Key")
```

Comment:
The sync hello_world deletes both the rsa key and the ec key, but the async version only deletes the rsa key